src/main/java/seedu/taassist/model/ModelManager.java

```diff
@@ -131,6 +133,21 @@ public boolean hasModuleClass(ModuleClass moduleClass) {
     public void deleteModuleClass(ModuleClass target) {
         requireNonNull(target);
         taAssist.removeModuleClass(target);
+
+        // TODO: Should an Exception be thrown instead?
```

Comment:
Do you mean that since the user is deleting the module class that is in focus mode, hence it should not be deleted and an exception should be thrown to warn one that the class is in focus mode now?

Reply:
I think it is fine for now. However, in the future we may consider showing an alert before any delete command to warn the user.